CHANGELOG.md

```diff
@@ -4,6 +4,64 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-beta.4](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.4) - 2022-01-26
+
+Zebra's latest beta improves the networking code and fixes some bugs. A couple of fixed bugs had
+caused Zebra to hang in some situations. Some improvements to the documentation were also included.
```

Comment:
Do we want to mention Rust 2021 edition here?

Reply:
Done.